src/lib/Command/IbexaSolrCommand.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Platform\PostInstall\Command;
+
+use Composer\Command\BaseCommand;
+use Composer\IO\IOInterface;
+use Ibexa\Platform\PostInstall\IbexaProductVersion;
+use Ibexa\Platform\PostInstall\VersionDirectoryProvider;
+use InvalidArgumentException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\ProcessBuilder;
+
+/**
+ * @internal
+ */
+class IbexaSolrCommand extends BaseCommand
```

Comment:
Naming: It needs to be (`IbexaSetupSolrCommand`) and should probably be `final`.
```suggestion
final class IbexaSetupSolrCommand extends BaseCommand
```